### PR TITLE
hoc2021: code.org/ homepage for CSC gets wider

### PR DESCRIPTION
**Pull request by breville**
This supports widths of up to 1920px.

Follow-up to https://github.com/code-dot-org/code-dot-org/pull/43554.

### desktop

localhost code org_3000_ (3)

### mobile

localhost code org_3000_ (4)

### CODE REVIEW

**Review by KylieModen — APPROVED** [review]
lgtm

**Review by maddiedierker — APPROVED** [review]

**Comment by maddiedierker** [comment]
just a heads up to @breville -- if the build on this succeeds before you're online today, i'll merge so we can ship it with today's HoC-launch-DTP. otherwise, i think it's fine for this to ship tomorrow!

**Pull request merged**
